packages/components/scripts/generate-react-components.js

```diff
@@ -0,0 +1,90 @@
+const { checkFolderExist } = require("../../../scripts/helper");
+const fs = require("fs");
+const config = require("../../icons/scripts/svgo-config");
+const { transform } = require("@svgr/core");
+const camelCase = require("camelcase");
+const { groupBy } = require("lodash");
+
+const GENERATED_HEADER = `/*
+* THIS FILE IS GENERATED. DO NOT EDIT IT.
+* To re-generate the icons, run the yarn generate-icons command in the components project.
+*/
+`;
+
+const svgTemplate = content => `${GENERATED_HEADER}
+/* eslint-disable */
+${content}
+/* eslint-enable */
+`;
+
+const indexFileTemplate = (iconNames, groupedIcons) => `${GENERATED_HEADER}
+/* eslint-disable */
+import { createOrbitIcon, createOrbitMultiVariantIcon } from "../createOrbitIcon";
+
+${iconNames.map(icon => `import { ReactComponent as Inner${icon} } from "./${icon}";`).join("\n")}
+
+${iconNames.map(icon => `export const ${icon} = createOrbitIcon(Inner${icon}, "${icon}");`).join("\n")}
+
+${Object.entries(groupedIcons).map(([key, group])=> {
+        const name = getComponentName(key);
+
+        return `export const ${name} = createOrbitMultiVariantIcon(${group.map(icon => `Inner${getComponentName(icon.name, icon.sizeInName)}`).join(", ")}, "${name}");`;
+    }).join("\n")}
```

Comment:
Indentation issue

Reply:
Not sure what the issue your seeing is

Reply:
![image](https://user-images.githubusercontent.com/794579/161339006-d012424f-d8bd-4288-b9f7-16317a15d00b.png)